src/components/UserMenu.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div>
+    <oc-button id="_userMenuButton" class="oc-topbar-personal uk-height-1-1" variation="primary" aria-label="$gettext('User Menu')" ref="menuButton">
+      <avatar type="span" class="oc-topbar-personal-avatar" :userid="userId" />
+      <span class="oc-topbar-personal-label">{{ userDisplayName }}</span>
+    </oc-button>
+    <oc-drop toggle="#_userMenuButton" mode="click" :options="{pos:'bottom-right'}" class="uk-width-large" ref="menu">
+      <div class="uk-card-body uk-flex uk-flex-middle uk-flex-column">
+        <avatar
+          :userid="userId"
+          :width="128"
+        />
+        <h3 class="uk-card-title">{{ userDisplayName }}</h3>
+        <span v-if="userEmail">{{ userEmail }}</span>
+        <router-link to="/account" target="_blank"><translate>Manage your account</translate></router-link>
+        <br/>
+        <oc-button id="logoutMenuItem" type="a" @click="logout()"><translate>Log out</translate></oc-button>
+      </div>
+      <div class="uk-card-footer uk-flex uk-flex-middle uk-flex-column">
+        <span>Version: {{appVersion.version}}-{{appVersion.hash}} ({{appVersion.buildDate}})</span>
```

Comment:
Looking now at the version... what about dropping the hash and buildDate when we started doing the releases? I understand that it made sense when we had it before them but now I'd say version should be enough.

Reply:
only the version helps it you have a proper deployment

But I agree that it might look not that nice - we could hide hash and date somewhere ...

Reply:
let's keep it for now as we're still in beta mode

once we talk about production, we could have debug mode vs production mode, where debug would display the hash